libs/labelbox/tests/integration/test_project_model_config.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+def test_add_single_model_config(configured_project, model_config):
+    project_model_config_id = configured_project.add_model_config(model_config.uid)
+
+    assert set(config.uid for config in configured_project.project_model_configs()) == set([project_model_config_id])
+
+    configured_project.remove_project_model_config(project_model_config_id)
+
+
+def test_add_multiple_model_config(client, rand_gen, configured_project, model_config, valid_model_id):
+    second_model_config = client.create_model_config(rand_gen(str), valid_model_id, {"param": "value"})
+    first_project_model_config_id = configured_project.add_model_config(model_config.uid)
+    second_project_model_config_id = configured_project.add_model_config(second_model_config.uid)
+    expected_model_configs = set([first_project_model_config_id, second_project_model_config_id])
+
+    assert set(config.uid for config in configured_project.project_model_configs()) == expected_model_configs
+
+    [configured_project.remove_project_model_config(project_model_config_id) for project_model_config_id in expected_model_configs]
+
+
+def test_remove_model_config_from_project(configured_project, model_config):
+    assert configured_project.remove_project_model_config(configured_project.add_model_config(model_config.uid))
+
+def test_delete_project_model_config(configured_project, model_config):
+    assert configured_project.remove_project_model_config(configured_project.add_model_config(model_config.uid))
```

Comment:
would be nice to retrieve back configs to make sure they have been indeed removed

Reply:
Sure, done.

Reply:
users can see project model configs are gone with the project.project_model_configs command. I added get_model_config to the client

Reply:
Actually not adding this in this PR, but will do in a followup.